lib/builder/webpack/base.js

```diff
@@ -75,9 +75,9 @@ export default class WebpackBaseConfig {
       'process.mode': JSON.stringify(this.options.mode),
       'process.static': this.isStatic
     }
-    _.each(this.options.env, (value, key) => {
+    Object.entries(this.options.env).forEach(([key, value]) => {
```

Comment:
Is `for...of Object.entries` better ?

Reply:
Not sure if it makes a huge difference :thinking: 
Thinking about expressiveness I'd guess it comes to the same.